triggermail_templates.py

```diff
@@ -128,6 +134,53 @@ def dissect_filename(self, template_filename):
         self.partner = self.settings.get("partner", self.partner) or self.partner
         self.partner = self.partner.replace("_templates", "")
 
+    def generate_pertinent_file_names_and_map(self, template_filename):
+        template_filename = template_filename.replace(self.path + '/', '')
```

Comment:
Is this trying to get just the file name part of this? It seems a bit weird to me to use `.replace`, since that implies that "self.path" might appear somewhere at the beginning. Would `os.path.basename` do what you want?